generator/source/faq/rest.rst

```diff
@@ -116,6 +116,10 @@ Back pressure is a flow-control technique to help prevent DoS attacks.
 It results in a ``Status.QUEUE_FULL`` client batch submit response or a 429 "Too Many Requests" REST API error.
 If the validator is overwhelmed it will stop accepting new batches until it can handle more work. The number of batches that validator can accept is based on a multiplier,  QUEUE_MULTIPLIER (currently 10, formerly 2), times a rolling average of the number of published batches.
 
+Can I disable the back pressure test?
+-------------------------------------
+No. There isn't a way to disable that currently because it's determined based on a multiplier of the publishing rate of the network. It allows the slower members of the network to not drift behind the faster members when it comes to staying in consensus. You would have to make a custom build of Sawtooth to remove that check.
```

Comment:
Accepted. Reworded answer in _italics_:

> No. There isn't a way to disable that currently because it's determined based on a multiplier of the publishing rate of the network. _Back pressure test is about rejecting load (in the form of batches) that the network wouldn't be able to accommodate in a reasonable amount of time (next couple blocks). This should never be turned off as without this feature it is trivial to do a client-based attack which overwhelms the network creating a DoS_. You would have to make a custom build of Sawtooth to remove that check.